### ISSUE TITLE
Can't deploy filebeat charm

### ISSUE THREAD

**Issue by cmars** [juju-solutions/layer-filebeat#63]
filebeat-19 is failing to deploy on xenial with the following errors in the unit log:

```
2018-10-16 17:00:54 INFO juju-log Invoking reactive handler: ../../../../../../usr/local/lib/python3.5/dist-packages/charmhelpers/core/host.py:704:wrapped_f
2018-10-16 17:00:54 INFO juju-log status-set: waiting: Waiting for: elasticsearch, logstash or kafka.
2018-10-16 17:00:55 DEBUG beats-host-relation-joined /var/lib/juju/agents/unit-filebeat-6/charm/hooks/beats-host-relation-joined: line 8: ../.venv/bin/chlp: No such file or directory
2018-10-16 17:00:55 ERROR juju.worker.uniter.operation runhook.go:114 hook "beats-host-relation-joined" failed: exit status 127
2018-10-16 17:01:01 DEBUG beats-host-relation-joined /var/lib/juju/agents/unit-filebeat-6/charm/hooks/beats-host-relation-joined: line 8: ../.venv/bin/chlp: No such file or directory
2018-10-16 17:01:01 ERROR juju.worker.uniter.operation runhook.go:114 hook "beats-host-relation-joined" failed: exit status 127
2018-10-16 17:01:12 DEBUG beats-host-relation-joined /var/lib/juju/agents/unit-filebeat-6/charm/hooks/beats-host-relation-joined: line 8: ../.venv/bin/chlp: No such file or directory
2018-10-16 17:01:12 ERROR juju.worker.uniter.operation runhook.go:114 hook "beats-host-relation-joined" failed: exit status 127
2018-10-16 17:01:33 DEBUG beats-host-relation-joined /var/lib/juju/agents/unit-filebeat-6/charm/hooks/beats-host-relation-joined: line 8: ../.venv/bin/chlp: No such file or directory
2018-10-16 17:01:33 ERROR juju.worker.uniter.operation runhook.go:114 hook "beats-host-relation-joined" failed: exit status 127
2018-10-16 17:02:17 DEBUG beats-host-relation-joined /var/lib/juju/agents/unit-filebeat-6/charm/hooks/beats-host-relation-joined: line 8: ../.venv/bin/chlp: No such file or directory
2018-10-16 17:02:17 ERROR juju.worker.uniter.operation runhook.go:114 hook "beats-host-relation-joined" failed: exit status 127
2018-10-16 17:03:40 DEBUG beats-host-relation-joined /var/lib/juju/agents/unit-filebeat-6/charm/hooks/beats-host-relation-joined: line 8: ../.venv/bin/chlp: No such file or directory
2018-10-16 17:03:40 ERROR juju.worker.uniter.operation runhook.go:114 hook "beats-host-relation-joined" failed: exit status 127
2018-10-16 17:06:22 DEBUG beats-host-relation-joined /var/lib/juju/agents/unit-filebeat-6/charm/hooks/beats-host-relation-joined: line 8: ../.venv/bin/chlp: No such file or directory
2018-10-16 17:06:22 ERROR juju.worker.uniter.operation runhook.go:114 hook "beats-host-relation-joined" failed: exit status 127
2018-10-16 17:11:23 DEBUG beats-host-relation-joined /var/lib/juju/agents/unit-filebeat-6/charm/hooks/beats-host-relation-joined: line 8: ../.venv/bin/chlp: No such file or directory
2018-10-16 17:11:23 ERROR juju.worker.uniter.operation runhook.go:114 hook "beats-host-relation-joined" failed: exit status 127
2018-10-16 17:16:23 DEBUG beats-host-relation-joined /var/lib/juju/agents/unit-filebeat-6/charm/hooks/beats-host-relation-joined: line 8: ../.venv/bin/chlp: No such file or directory
2018-10-16 17:16:23 ERROR juju.worker.uniter.operation runhook.go:114 hook "beats-host-relation-joined" failed: exit status 127
2018-10-16 17:21:24 DEBUG beats-host-relation-joined /var/lib/juju/agents/unit-filebeat-6/charm/hooks/beats-host-relation-joined: line 8: ../.venv/bin/chlp: No such file or directory
2018-10-16 17:21:24 ERROR juju.worker.uniter.operation runhook.go:114 hook "beats-host-relation-joined" failed: exit status 127
2018-10-16 17:26:24 DEBUG beats-host-relation-joined /var/lib/juju/agents/unit-filebeat-6/charm/hooks/beats-host-relation-joined: line 8: ../.venv/bin/chlp: No such file or directory
2018-10-16 17:26:24 ERROR juju.worker.uniter.operation runhook.go:114 hook "beats-host-relation-joined" failed: exit status 127
```

**Comment by johnsca** [comment]
For reference, it seems that the beats-base layer is overriding that one hook: https://github.com/juju-solutions/layer-beats-base/tree/master/hooks

It also seems that this is entirely unnecessary and is probably a hold-over from older code: https://github.com/juju-solutions/layer-beats-base/blob/master/lib/elasticbeats.py#L77

**Comment by kwmonroe** [comment]
Should be fixed in rev20 by https://github.com/juju-solutions/layer-beats-base/pull/32:

https://jujucharms.com/filebeat/20

**Comment by cmars** [comment]
Fixed by juju-solutions/layer-beats-base#32

**Comment by cmars** [comment]
Thanks @kwmonroe !
